R/common.R

```diff
@@ -2881,8 +2881,21 @@ editImage <- function(optionsJson) {
   if (is.list(options) && c("setSeed", "seed") %in% names(options)) {
     if (isTRUE(options[["setSeed"]]))
       set.seed(options[["seed"]])
+  } else {
+    stop(paste(".setSeedJASP was called with an incorrect argument.",
+               "The argument options should be the options list from QML.",
+               "Ensure that the SetSeed{} QML component is present in the QML file for this analysis."))
+  }
+}
+
+.getSeedJASP <- function(options) {
+  
+  if (is.list(options) && c("setSeed", "seed") %in% names(options)) {
+    if (isTRUE(options[["setSeed"]]))
+      return(options[["seed"]])
+  } else {
+    stop(paste(".getSeedJASP was called with an incorrect argument.",
```

Comment:
I think this will pose a problem for functions that are shared between analyses (or even modules) when one analysis defines the seed component in the qml interface and the other does not. I don't know if such shared functions still exist, but we should be sure before uncommenting this.

Reply:
oh, true. I forget that we have something like that in the t-tests :/ not great, I will comment it out again then

Reply:
Could you also add a comment explaining why the code is commented out? Will probably be useful in the future

Reply:
Note that there are two of these `stop()`'s